app/views/sites/twitter_profiles/new.html.haml

```diff
@@ -11,6 +11,7 @@
   = render_error_messages(@profile)
   = f.label :screen_name, 'Twitter Handle', class: 'required'
   = f.text_field :screen_name, class: 'required input-primary'
-  = label_tag nil, class: 'checkbox' do
-    = check_box_tag :show_lists
-    Show tweets from my lists
+  / Temporarily disabling checkbox as per JIRA ticket #SRCH-3556
```

Comment:
This is a preference call so I won't block on it, but I'd personally change both these comments and those in `app/views/sites/twitter_profiles/index.html.haml` from an html-style comment (`/`) to a ruby comment (`-#`).  That way the comments won't be visible to users in the html of the page/sent to the client, which they probably don't need to be.

Reply:
Updated the suggested changes above